client/src/components/BaseInput.vue

```diff
@@ -19,6 +19,7 @@
       @blur="isFocused = false"
     />
     <label v-if="label && !showLabelOnTop">{{ label }}</label>
+    <!-- TODO: Remove or improve? -->
```

Comment:
Gute Frage.
Warum haben wir/ich das damals nur auskommentiert?
Fände es eigentlich besser, wenn wir den Validation Text in der Komponente direkt rendern.
So musstest du ja die Logik für dieses Label wieder in UserProfileDetails implementieren.

Reply:
Die Logik muss ja eh woanders rein, ich glaube du hattest das einfach mal angefangen. Das wäre dann eher ein PR für sich.